### DISCUSSION TITLE
Classes Panel, Stack and Split are raising not found error

### DISCUSSION THREAD

**Discussion by nrep** [filamentphp/filament#4186] [General]
![Screenshot from 2022-09-24 21-44-44](https://user-images.githubusercontent.com/10692907/192115855-8f63676d-9454-4c1b-b0d2-ac6ea136ccb4.png)
What can I do to use the Panel, Split and Stack

**Comment by Z3d0X** [comment]
Please upgrade to the latest Filament version using `composer update`. That feature was released only a couple days ago

**Reply by nrep** [reply]
Thanks. It worked.
